modules/ROOT/pages/configuring-dev-env.adoc

```diff
@@ -136,11 +136,49 @@ init(aerogearConfig);
 
 . Select `AeroGear.Mobile.Core` and depending on your target platforms:
 
-.. for Android also pick `AeroGear.Mobile.Core.Platform.Android`
+. for Android:
 
-.. for iOS also pick `AeroGear.Mobile.Core.Platform.iOS`
+.. also pick `AeroGear.Mobile.Core.Platform.Android`
 
-. Click on `Add Packages`
+.. Click on `Add Packages`
+
+.. Initialize Core (for Xamarin.Forms):
++
+[souce,csharp]
+----
+var app = new App();
+MobileCoreAndroid.Init(app.GetType().Assembly,ApplicationContext);
+LoadApplication(app);
+----
+
+.. Initialize Core (for Legacy Xamarin):
++
+[souce,csharp]
+----
+MobileCoreAndroid.Init(ApplicationContext);
+----
+
+. for iOS:
+
+.. also pick `AeroGear.Mobile.Core.Platform.iOS`
+
+.. Click on `Add Packages`
```

Comment:
I believe this shoulbe before the previous bullet

Reply:
Actually this refers to the `Add Packages` button in the dialog where you add the package. But it looks a bit confusing. I think i just remove the whole bullet point.